tests/components/powerwall/test_powerwall_data_manager.py

```diff
@@ -0,0 +1,51 @@
+"""Tests for the PowerwalLDataManager."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+from tesla_powerwall import AccessDeniedError, Powerwall
+
+from homeassistant.components.powerwall import PowerwallDataManager
+from homeassistant.components.powerwall.models import PowerwallRuntimeData
+from homeassistant.exceptions import ConfigEntryAuthFailed
+
+
```

Comment:
Let's move this into `test_init.py` so it matches the naming of the code we are testing.